share/doc/wake/basic_wake_tutorial.md

```diff
@@ -0,0 +1,628 @@
+## Wake Tutorial
+
+### Invoking wake
+
+1. Create a wake database: `wake --init .`
+
+2. Run command for wake: `wake -vx '<function_name> <parameter_if_any>'`
```

Comment:
is there a `wake --help` to see all the options? What does -vx mean?

Reply:
I will add this info in the next commit